### PR TITLE
Chore: Bump node version to 22

### PR DESCRIPTION
**Pull request by bensleveritt**

### CODE REVIEW

File: packages/cojson-storage-sqlite/package.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
   "types": "src/index.ts",
   "license": "MIT",
   "dependencies": {
-    "better-sqlite3": "^8.5.2",
+    "better-sqlite3": "^11.7.0",
```
Comment (gdorsi) [inline packages/cojson-storage-sqlite/package.json:9]: This means that we are dropping support for node v21: https://github.com/WiseLibs/better-sqlite3/releases/tag/v11.0.0

**Review by gdorsi — APPROVED** [review]
LGTM

Thanks!

File: packages/cojson-storage-sqlite/package.json (hunk 1 above)
Comment (bensleveritt) [reply packages/cojson-storage-sqlite/package.json:9]: Good point. Is that a concern, or should we try to stay under?

**Review by gdorsi — APPROVED** [review]
Many thanks!

**Pull request merged**